js/adapt-contrib-slider.js

```diff
@@ -383,15 +383,15 @@ define([
             this.$('.slider-markers').empty();
             if (this.model.get('_showScale') === false) {
                 this.$('.slider-markers').eq(0).css({display: 'none'});
-                this.model.get('_showScaleIndicator')
-                    ? this.$('.slider-scale-numbers').eq(0).css({visibility: 'hidden'})
-                    : this.$('.slider-scale-numbers').eq(0).css({display: 'none'});
+                this.$('.slider-scale-numbers').eq(0).css(
+                    this.model.get('_showScaleIndicator') ? {visibility: 'hidden'} : {display: 'none'}
+                );
             } else {
                 var $scaler = this.$('.slider-scaler');
                 var $markers = this.$('.slider-markers');
                 for (var i = 0, count = this.model.get('_items').length; i < count; i++) {
                     $markers.append("<div class='slider-line component-item-color'>");
-                    $('.slider-line', $markers).eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
+                    this.$('.slider-line', $markers).eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
```

Comment:
The selector is the string which forms the first argument. The second argument, which denotes the context in which the selector is performed.